apps/src/lib/levelbuilder/rubrics/RubricsContainer.jsx

```diff
@@ -20,32 +20,32 @@ export default function RubricsContainer({
       ? rubric.learningGoals
       : [
           {
-            key: 'learningGoal-1',
-            id: 'learningGoal-1',
+            key: 'ui-1',
+            id: 'ui-1',
             learningGoal: '',
             aiEnabled: false,
             position: 1,
             learningGoalEvidenceLevelsAttributes: [
               {
-                learningGoalId: 'learningGoal-1',
+                // learningGoalId: 'learningGoal-1',
```

Comment:
nit: remove commented out code

Reply:
On my side, this is outdated code.  It is removed now!  Let me know if I need to change any settings or re-request review on the most up-to-date code somehow. 

<img width="957" alt="image" src="https://github.com/code-dot-org/code-dot-org/assets/24883357/3aaab557-bc05-4150-a748-548d7bc75506">